backend/app/cmd/server.go

```diff
@@ -473,22 +473,24 @@ func (s *ServerCommand) newServerApp(ctx context.Context) (*serverApp, error) {
 	}
 
 	var emailNotifications bool
-	notifyService, telegramBotUsername, err := s.makeNotify(dataService, authenticator)
-
-	if contains("email", s.Notify.Users) {
-		emailNotifications = true
-	}
-
-	// we pass telegramBotUsername to Rest server only if user notifications are enabled
-	if !contains("telegram", s.Notify.Users) {
-		telegramBotUsername = ""
-	}
+	var telegramBotUsername string
+	notifyService, telegramService, err := s.makeNotify(dataService, authenticator)
 
 	if err != nil {
```

Comment:
even one level of if/else makes me uncomfortable but here we have nested if/else. I'd suggest to simplify it and get rid of else with an anonymous or real function with early exit(s)